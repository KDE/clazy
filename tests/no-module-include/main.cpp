#include <QtCore/QtCore>
#include <QtCore>
#include <QtGui/QtGui>
#include <QtGui>
#include <QtQml/QtQml>
#include <QtQml>
#include <QtTest/QtTest>
#include <QtTest>
#include <QtNetwork/QtNetwork>
#include <QtNetwork>
#include <QtDBus/QtDBus>
#include <QtDBus>
#include <QtQuick/QtQuick>
#include <QtQuick>
#include <QtQmlModels/QtQmlModels>
#include <QtQmlModels>
#include <QtXml/QtXml>
#include <QtXml>
#include <QtSvg/QtSvg>
#include <QtSvg>
#include <QtConcurrent/QtConcurrent>
#include <QtConcurrent>
#include <QtMultimedia/QtMultimedia>
#include <QtMultimedia>
#include <QtSql/QtSql>
#include <QtSql>
#include <QtPrintSupport/QtPrintSupport>
#include <QtPrintSupport>
#include <QtNetworkAuth/QtNetworkAuth>
#include <QtNetworkAuth>
#include <QtQuickTest>
#include <QtQuickTest/QtQuickTest>




