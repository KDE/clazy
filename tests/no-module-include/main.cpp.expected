no-module-include/main.cpp:1:10: warning: Module QtCore should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:2:10: warning: Module QtCore should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:3:10: warning: Module QtGui should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:4:10: warning: Module QtGui should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:5:10: warning: Module QtQml should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:6:10: warning: Module QtQml should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:7:10: warning: Module QtTest should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:8:10: warning: Module QtTest should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:9:10: warning: Module QtNetwork should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:10:10: warning: Module QtNetwork should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:11:10: warning: Module QtDBus should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:12:10: warning: Module QtDBus should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:13:10: warning: Module QtQuick should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:14:10: warning: Module QtQuick should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:15:10: warning: Module QtQmlModels should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:16:10: warning: Module QtQmlModels should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:17:10: warning: Module QtXml should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:18:10: warning: Module QtXml should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:19:10: warning: Module QtSvg should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:20:10: warning: Module QtSvg should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:21:10: warning: Module QtConcurrent should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:22:10: warning: Module QtConcurrent should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:23:10: warning: Module QtMultimedia should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:24:10: warning: Module QtMultimedia should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:25:10: warning: Module QtSql should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:26:10: warning: Module QtSql should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:27:10: warning: Module QtPrintSupport should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:28:10: warning: Module QtPrintSupport should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:29:10: warning: Module QtNetworkAuth should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:30:10: warning: Module QtNetworkAuth should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:31:10: warning: Module QtQuickTest should not be included directly [-Wclazy-no-module-include]
no-module-include/main.cpp:32:10: warning: Module QtQuickTest should not be included directly [-Wclazy-no-module-include]
